config/default/sitenow_pages.settings.yml

```diff
@@ -1,4 +1,4 @@
 featured_image_display_default: large
 tag_display: do_not_display
 related_display: do_not_display
-show_teaser_link_indicator: false
+show_teaser_link_indicator: true
```

Comment:
Just checking why this was changed, since it seems like an unrelated change.

Also, it looks like this is missing the config value that is added in `sitenow_pages.settings.yml` inside the module config folder below.